cms/djangoapps/contentstore/views/tests/test_import_export.py

```diff
@@ -427,3 +427,18 @@ def test_library_export(self):
             self.assertEqual(video_xml.get('youtube_id_1_0'), youtube_id)
         finally:
             shutil.rmtree(root_dir / name)
+
+    def test_export_success_with_custom_tag(self):
+        """
+        Verify that course export with customtag
+        """
+        xml_string = '<impl>slides</impl>'
+        vertical = ItemFactory.create(parent_location=self.course.location,
```

Comment:
nit: please use the indentation we ask for in the style guide: https://github.com/edx/edx-platform/wiki/Python-Guidelines#breaking-long-lines